packages/embark/src/cmd/cmd_controller.js

```diff
@@ -221,6 +229,14 @@ class EmbarkController {
       function initEngine(callback) {
         engine.init({}, callback);
       },
+      function (callback) {
+        engine.startService("libraryManager").installAll((err) => {
```

Comment:
Can replace this by `engine.startService("libraryManager").installAll(callback)`
Same for others

Reply:
Good catch, will do.

Reply:
So, I made the change and then saw in CI tests were failing. It *should* definitely work. I poked at it for a bit and I'm 99.9% confident that it's a bug in the `async` package. Unfortunately, v2.6.1 is the last in the 2.x series of `async` releases, and that's what we're using. I tried previously to upgrade embark to `async` 3.x — it breaks for embark for non-obvious reasons and I couldn't figure out how to fix it.

For now I've switched to this:
```
function (callback) {
  engine.startService("libraryManager").installAll((err) => callback(err ? err : null));
}
```

It works, even if it's more verbose.

Reply:
It's probably not a bug in async. 
I'm guessing one of the `installAll`s called back with a result (eg `cb(null, something`), so it broke the next function in the waterfall, because the arg we expected to be a function is now the result

Reply:
Ah, exactly right! Good catch. I wasn't thinking about how in the waterfall non-error return values can be passed down, since we don't do much of that in cmd_controller.